docs/resources/aviatrix_aws_tgw_directconnect.md

```diff
@@ -18,7 +18,7 @@ resource "aviatrix_aws_tgw_directconnect" "test_aws_tgw_directconnect" {
   tgw_name                   = "my-aws-tgw-1"
   directconnect_account_name = "username"
   dx_gateway_id              = "30321d76-dd01-49bf"
-  security_domain_name       = "my-sdn-1"
+  network_domain_name        = "my-sdn-1"
```

Comment:
nitpicking, but the actual value here still says 'sdn' , which would refer to 'security domain'